### PR TITLE
add dev docker setup for lix, fix missing lix origin warning logic

### PR DESCRIPTION
**Pull request by janfjohannes**

### CODE REVIEW

File: lix/packages/client/src/openRepository.ts [hunk 1]
```diff
@@ -126,13 +126,16 @@ export async function openRepository(
 		}
 	}
 
-	const { protocol, lixHost, repoHost, owner, repoName, username, password } = parseLixUri(url)
+	const { protocol, lixHost, repoHost, owner, repoName, username, password, namespace } =
+		parseLixUri(url)
```
Comment (martin-lysk) [inline lix/packages/client/src/openRepository.ts:130]: lets document parseLixUri a bit better together later on

File: lix/packages/client/src/openRepository.ts [hunk 2]
```diff
@@ -126,13 +126,16 @@ export async function openRepository(
 		}
 	}
 
-	const { protocol, lixHost, repoHost, owner, repoName, username, password } = parseLixUri(url)
+	const { protocol, lixHost, repoHost, owner, repoName, username, password, namespace } =
```
[the remark this answers is not in the document]
Comment (janfjohannes) [reply lix/packages/client/src/openRepository.ts:131]: @martin-lysk  that is on purpose as lxi currently has to handle weird edgeaces with invalid lix uris and local repos, logging the error got complaints

File: lix/packages/client/src/openRepository.ts (hunk 1 above)
Comment (martin-lysk) [reply lix/packages/client/src/openRepository.ts:132]: asadadas

Comment (martin-lysk) [reply lix/packages/client/src/openRepository.ts:132]: asasdasd

**Pull request merged**